MVPproject/MVPproject/RecipesModule/View/SortButtonsView.swift

```diff
@@ -0,0 +1,123 @@
+// SortButtonsView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+/// Протокол для делегата SortButtonsView
+protocol SortButtonViewDelegate: AnyObject {}
+
+/// Вью с 2мя кнопками сортировки
+final class SortButtonsView: UIView {
+    // MARK: - Visual components
+
+    private let caloriesButton: UIButton = {
+        let button = UIButton(type: .system)
+        button.setTitle("Calories  ", for: .normal)
+        button.setImage(.stackUp, for: .normal)
+        button.semanticContentAttribute = .forceRightToLeft
+        button.backgroundColor = .systemGray4
+        button.layer.cornerRadius = 12
+        button.tintColor = .black
+        return button
+    }()
+
+    private let timeButton: UIButton = {
+        let button = UIButton(type: .system)
+        button.setTitle("Time  ", for: .normal)
+        button.setImage(.stackUp, for: .normal)
+        button.semanticContentAttribute = .forceRightToLeft
+        button.backgroundColor = .systemGray4
+        button.layer.cornerRadius = 12
+        button.tintColor = .black
+        return button
+    }()
+
+    private lazy var buttonsStackView: UIStackView = {
+        let stack = UIStackView(arrangedSubviews: [caloriesButton, timeButton])
+        stack.axis = .horizontal
+        stack.distribution = .fill
+        stack.spacing = 11
+        stack.disableAutoresizingMask()
+        timeButton.widthAnchor.constraint(equalTo: caloriesButton.widthAnchor, multiplier: 0.8).isActive = true
```

Comment:
я еще и activate() добавила, но это тоже как тебе удобнее
```suggestion
        timeButton.widthAnchor.constraint(equalTo: caloriesButton.widthAnchor, multiplier: 0.8).activate()
```